cypress/support/elements/researcher_anonymous_classes_elements.js

```diff
@@ -0,0 +1,146 @@
+		this.getResultsTable().find('tbody tr').eq(0).find('[class^=linkCell--] a').should("exist");
+	},
+	getResultsTableSchoolColumn() {
+		return this.getResultsTable().find('thead tr th').eq(3);
+	},
+	verifySchoolColumnSortIcon() {
+		this.getResultsTableSchoolColumn().find('.icon-sort').should("exist");
+	},
+	clickSortIcon(column) {
+		this.getResultsTable().find('thead tr th').eq(column).find('.icon-sort').click();
+	},
+	verifyFirstRowData(column, data) {
+		this.getResultsTable().find('tbody tr').eq(0).find('td').eq(column).should("contain", data);
+	},
+	getResetAllButton() {
+    return cy.get('#form-container#form-container [class^=bottom--] [class^=summary--] button');
+	},
+  verifySummary() {
+    cy.get('#form-container#form-container [class^=bottom--] [class^=summary--]').should("contain", "Your filter matches: ");
+	},
+  verifySummaryNotDisplayed() {
+    cy.get('#form-container#form-container [class^=bottom--] [class^=summary--]').should("not.exist");
+	},
+	verifyConcordConsortiumSchoolNameDisplayed() {
+		this.getResultsTable().find('tbody tr').eq(0).find('td').eq(3).should("contain", "Concord Consortium");
+	},
+	verifyConcordConsortiumSchoolNameNotDisplayed() {
+		this.getResultsTable().find('tbody tr').eq(0).find('td').eq(3).should("not.contain", "Concord Consortium");
+	},
+
+  // Research classes landing page
+
+  getAssignmentPage() {
+    return cy.get('#assignments-page');
+  },
+  verifyHeader() {
+    this.getAssignmentPage().find('[class^=classAssignments--] header').should("contain", "Automation Teacher 3 Class");
+  },
+  getOfferingsTable() {
+    return this.getAssignmentPage().find('[class^=offeringsTable--]');
+  },
+  getFirstOffering() {
+    return this.getOfferingsTable().find('[class^=offering--]').eq(0);
+  },
+  verifySortIconNotDisplayed() {
+    this.getFirstOffering().find('.icon-sort').should("not.exist");
+  },
+  verifyActiveCheckboxDisabled() {
+    this.getFirstOffering().find('[class^=checkboxCell--] input').eq(0).invoke("attr", "disabled").should("exist");
+  },
+  verifyLockedCheckboxDisabled() {
+    this.getFirstOffering().find('[class^=checkboxCell--] input').eq(1).invoke("attr", "disabled").should("exist");
+  },
+  clickShowDetails() {
+    this.getFirstOffering().find('[class^=detailsCell--] button').click();
+  },
+  verifyOfferingButtons() {
+    this.getFirstOffering().find('[class^=offeringDetails-] a')
+    .should("contain", "Preview")
+    .should("contain", "Class Dashboard");
+  },
+  verifyAnonymousStudentName() {
+    this.getFirstOffering().find('[class^=offeringProgress--] [class^=name--]')
+    .should("contain", "Student 3")
+    .should("contain", "Student 33")
+    .should("contain", "Student 4");
+  },
+
+  // Expiration date
+
+  getResearcherProject() {
+    return cy.get('.options-list .inline-fields label').contains("Test Project For Researcher").parent();
+  },
+  getResearcherProjectChecbox() {
```

Comment:
Misspelled: "Checbox" should be "Checkbox".

Reply:
Fixed